cmd/print.go

```diff
@@ -263,6 +262,7 @@ if specified with --namespace.`)
 		}
 	}
 
+	_ = cmd.MarkFlagRequired("providers")
```

Comment:
I think we can add validation that ensures openapi is the only provider when set.

Reply:
done, PTAL!